x/auth/client/cli/query.go

```diff
@@ -121,6 +129,8 @@ $ %s query txs --%s 'message.sender=cosmos1...&message.action=withdraw_delegator
 `, eventFormat, version.AppName, flagEvents),
 		),
 		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx := client.GetClientContextFromCmd(cmd)
```

Comment:
any reason to avoid reading querycommand flags here?

Reply:
Because we don't call `AddQueryFlagsToCmd` on the command.